crates/settlement-clients/ethereum/src/lib.rs

```diff
@@ -69,7 +69,7 @@ impl EthereumSettlementClient {
             ProviderBuilder::new().with_recommended_fillers().wallet(wallet.clone()).on_http(settlement_cfg.rpc_url),
         );
         let core_contract_client = StarknetValidityContractClient::new(
-            Address::from_slice(settlement_cfg.core_contract_address.as_bytes()).0.into(),
+            Address::from_str(&settlement_cfg.core_contract_address).unwrap().0.into(),
```

Comment:
add comment on why unwrap is safe

Reply:
Alright, 
Added :
```
// It's safe to use unwrap here since this is `setup` code,
// if anything fails it is pre-assumed that any errors at this stage are handled in real time.
```


### IMPORTANT 
[This branch is a fork from [increasing coverage](https://github.com/madara-alliance/madara-orchestrator/tree/feat/increasing-test-coverage) ]
[This comment refers to changed already merged to main, they are shown here since #52 is merged and the base of this PR automatically shifted to `main` ]